app/src/routes/+page.svelte

```diff
@@ -31,17 +28,24 @@
 	<textarea bind:value={entered_text} rows="5" autofocus class="textarea textarea-bordered textarea-lg w-4/5" />
 </form>
 
-<div class="divider my-12" class:divider-success={success} class:divider-error={has_error}>
-	{#if success}
-		<Icon icon={`mdi:check-circle`} class="h-16 w-16 text-success" />
-	{:else if has_error}
-		<Icon icon={`mdi:close-circle`} class="h-16 w-16 text-error" />
-	{/if}
-</div>
+{#await tokens_promise}
```

Comment:
```suggestion
{#await parse(entered_text)}
```

Reply:
Since this file is so short and easy to read, the extra variable name is not helping that much here and it's not used anywhere else.